whitepapers/mechanisms/ExponentialMechForMedian.tex

```diff
@@ -38,25 +38,64 @@ \section{The Exponential Mechanism}
 
 Note that the exponential mechanism may not be tractable in many cases, as it assumes the existence of a utility function, and even if one exists it may not be tractable to compute it efficiently. 
 
-\section{An Exponential Mechanism for a Median}
+\section{An Exponential Mechanism for a quantile}
 
 \subsection{Defining a sensible utility function}
 
-Note that a user will prefer an output that is closer to the true median over one that is further away. Let $x$ be an (ordered) data set, let $r$ be a possible output, and let $n$ be the size of the data set. Note that if $r$ is close to the median, its rank will be close to $n/2$. Then, the following utility function is a sensical one:
+Note that a user will prefer an output that is closer to the true quantile over one that is further away. Let $x$ be an (ordered) data set, let $r$ be a possible output, and let $N$ be the size of the data set. Let $\#(Z>r)$ refer to the number of points in $x$ above $r$. Then, the following is a reasonable utility function for a release $r$ for the $\alpha$-quantile of $x$.
 
 \begin{equation}
-u(x, r) = - \left\vert \text{rank}_x(r) - \frac{n}{2} \right\vert.\footnote{Note that you could use a different measure of distance here if desired; an $L_1$ norm in terms of rank is just convenient to reduce sensitivity.}
-\end{equation}
-
-Note that the \textit{rank} function here is a slight abuse of notation since $r$ is an element from the \textit{data range}, not from the underlying database $x$, so does not have a rank. 
+u(x,r) = \max(\alpha, (1-\alpha))N - \vert (1-\alpha)\#(Z<r) - \alpha\#(Z>r)\vert.
+\end{equation} 
 
 \subsection{Sensitivity of the utility function}
 
 \subsubsection{Neighboring Definition: Change One}
-
+\begin{lemma}
+The above utility function $u$ has $\ell-1$ sensitivity bounded above by 1 in the change one model.
+\end{lemma}
+
+\begin{proof}
+Let $c_1 = \#(Z<r)$ and $c_2 = \#(Z>r)$. In the worst case, $c_1$ increases by 1 and $c_2$ decreases by 1.
```

Comment:
I think that the worst-case is either what you describe or the opposite ($c_1$ decreases by 1 and $c_1$ increases by 1). They both simplify to $\leq 1$ after the reverse triangle inequality, so I don't think you need to show a whole other proof or anything.

Reply:
Yeah; I did actually do it out once upon a time to triple check and it doesn't matter. Added a clarifying line.